.circleci/config.yml

```diff
@@ -105,17 +105,6 @@ jobs:
           https://github.com/go-redis/redis
           contrib/go-redis/redis/vendor/github.com/go-redis/redis
 
-    - run:
-        name: Vendor github.com/googleapis/gnostic for k8s.io/client-go
-        # This step checks out k8s.io/client-go and vendors
-        # github.com/googleapis/gnostic to fix a breaking change made in
-        # gnostic. See kubernetes/client-go#741
-        # TODO(knusbaum): remove this once the breaking change is resolved or propagated
-        command: |
-          git clone --branch v0.17.3 https://github.com/kubernetes/client-go $GOPATH/src/k8s.io/client-go
-          git clone --branch v0.17.3 https://github.com/kubernetes/apimachinery $GOPATH/src/k8s.io/apimachinery
-          git clone --branch v0.4.0 https://github.com/googleapis/gnostic $GOPATH/src/k8s.io/client-go/vendor/github.com/googleapis/gnostic
```

Comment:
It might be just this one line that needs to be removed. In my other PR I'm not getting this error and this one is missing.